zkevm-circuits/src/state_circuit.rs

```diff
@@ -554,8 +554,7 @@ fn queries<F: Field>(meta: &mut VirtualCells<'_, F>, c: &StateCircuitConfig<F>)
             field_tag: meta.query_advice(c.rw_table.field_tag, Rotation::cur()),
             storage_key: meta.query_advice(c.rw_table.storage_key, Rotation::cur()),
             value: meta.query_advice(c.rw_table.value, Rotation::cur()),
-            // TODO: we should constain value.prev() <-> value_prev.cur() later
-            // see https://github.com/privacy-scaling-explorations/zkevm-specs/issues/202 for more details
+            value_rotation_prev: meta.query_advice(c.rw_table.value, Rotation::prev()),
```

Comment:
Wait they are identical...these 2 lines

Reply:
that does explain why the tests passed so easily....